doc/contents/auditing_nova.rst

```diff
@@ -121,3 +121,20 @@ following to your ``top.nova`` file::
     nova:
       '*':
         - example_control.example
+
+It is quite possible you may want to send the NOVA audit results to a collection system. 
+This can be accomplished by using returners, which are included with Hubble. Currently, 
+there are returners available for **splunk**, **logstash**, **graylog**, and **sqlite**.
+Hubble supports using multiple returners from the same node so sending data to multiple 
+collectors is possible. Each individual returner has distinct options that need to be 
+set prior to their use. You can find general config options for each returner at the top 
+of each file located here
+https://github.com/hubblestack/hubble/tree/develop/hubblestack/extmods/returners.
+
+**Specify a returner when running Hubble from the command line:**::
+
+    hubble hubble.audit --r RETURN, --return RETURN
```

Comment:
Can we split this into two lines and add a `code-block` stanza?

```
.. code-block::

    hubble hubble.audit --r RETURN
    hubble hubble.audit --return RETURN
```

Reply:
Lines have been split and inserted into the code block. Thanks for the feedback.

Reply:
I actually missed the fact that you had the trailing `::` -- you actually had the code block piece right already! But this is more explicit, so I think it's still a good change. Thanks for making those changes!